gslab_scons/misc.py

```diff
@@ -89,34 +89,35 @@ def get_stata_executable(env):
     Windows cmd to carry a Stata batch job. 
 
     The function will check for user input in Scons env with
-    the flag e.g. `sf=StataMP-64.exe`. With no user input,
+    the flag e.g. `stata_executable=StataMP-64.exe`. With no user input,
     the function loops through common Unix and Windows executables
     and searches them in the system environment.
     '''
-    # Get environment's user input flavor. Empty default = None.
-    user_flavor  = env['user_flavor']  
+    # Get environment's user input executable. Empty default = None.
+    stata_executable  = env['stata_executable']  
 
-    if user_flavor is not None:
-        return user_flavor
+    if stata_executable is not None:
+        return stata_executable
     else:
-        flavors = ['stata-mp', 'stata-se', 'stata']
+        executables = ['stata-mp', 'stata-se', 'stata']
         if is_unix():
-            for flavor in flavors:
-                if is_in_path(flavor): # check in $PATH
-                    return flavor
+            for executable in executables:
+                if is_in_path(executable): # check in $PATH
+                    return executable
+
         elif sys.platform == 'win32':
             try:

```

Comment:
It will be clearer to rewrite this try/except block as an if/else using os.path.keys().